### PR TITLE
Sdk 92 - sipusers.

### PR DESCRIPTION
**Pull request by bednarz1990**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -278,3 +278,6 @@ Apidaze.SDK.Tests.Integration/UpdateExternalScriptUrl/client-secrets.json
 /Apidaze.SDK.Tests.Integration/SubApplicationExternalScriptsExample/client-secrets.json
 /Apidaze.SDK.Tests.Integration/ListApplications/client-secrets.json
 /Apidaze.SDK.Tests.Integration/GetApplicationActionExample/client-secrets.json
+/Apidaze.SDK.Tests.Integration/GetSipUsers/client-secrets.json
+Apidaze.SDK.Tests.Integration/CreateSipUserExample/client-secrets.json
```
Comment (shadzik) [inline .gitignore:282]: Somethings up here again

File: Apidaze.SDK.Tests.Integration/GetSipUsers/Program.cs [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+using System.IO;
+
+namespace GetSipUsersExample
+{
+    class Program
+    {
+        /// <summary>
+        /// Defines the entry point of the application.
+        /// </summary>
+        /// <param name="args">The arguments.</param>
```
Comment (shadzik) [inline Apidaze.SDK.Tests.Integration/GetSipUsers/Program.cs:15]: args are the arguments? No way Sherlock :) But what kind of arguments do you expect? :)

File: Apidaze.SDK/ApiActionFactory.cs [hunk 3]
```diff
@@ -28,7 +29,7 @@ internal class ApiActionFactory : IApiActionFactory
         /// </summary>
         /// <param name="credentials">The credentials.</param>
         /// <param name="url">The URL.</param>
-        internal ApiActionFactory(Credentials credentials, string url = "https://api4.apidaze.io/")
+        internal ApiActionFactory(Credentials credentials, string url = "https://cpaas-api.dev.voipinnovations.com/")
```
Comment (shadzik) [inline Apidaze.SDK/ApiActionFactory.cs]: Wrong URL, even if it's sandbox, we strive for production

File: Apidaze.SDK/ApplicationManager.cs [hunk 4]
```diff
@@ -14,7 +14,7 @@ public class ApplicationManager
         /// <param name="url">The URL.</param>
         /// <returns>IApiActionFactory.</returns>
         public static IApiActionFactory CreateApiFactory(Credentials credentials,
-            string url = "https://api.Apidaze.io/")
+            string url = "https://cpaas-api.dev.voipinnovations.com/")
```
Comment (shadzik) [inline Apidaze.SDK/ApplicationManager.cs]: Same here

File: Apidaze.SDK/CdrHttpHandlers/CdrHttpHandlers.cs [hunk 5]
```diff
@@ -40,78 +40,70 @@ public List<CdrHttpHandler> GetCdrHttpHandlers()
...
+            if (string.IsNullOrEmpty(name)) throw new ArgumentException("name must not be null or empty");
+            if (url == null) throw new ArgumentException("url must not be null or empty");
 
             var requestBody = new Dictionary<string, string> {{"name", name}, {"url", url.ToString()}};
             return Create<CdrHttpHandler>(requestBody);
         }
 
         /// <summary>
-        /// Updates the CDR HTTP handler.
+        ///     Updates the CDR HTTP handler.
```
Comment (shadzik) [inline Apidaze.SDK/CdrHttpHandlers/CdrHttpHandlers.cs:60]: Leading white spaces are better?

**Review by shadzik — CHANGES_REQUESTED** [review]

File: Apidaze.SDK/CdrHttpHandlers/CdrHttpHandlers.cs (hunk 5 above)
Comment (bednarz1990) [reply Apidaze.SDK/CdrHttpHandlers/CdrHttpHandlers.cs:60]: I think this is better for reading.

**Review by shadzik — APPROVED** [review]

**Review by barinali — APPROVED** [review]

**Pull request merged**